test/doc/algo/reverse.cpp

```diff
@@ -0,0 +1,18 @@
+#include <eve/module/core.hpp>
+#include <eve/algo.hpp>
+#include <iostream>
+#include <vector>
+#include "print.hpp"
+
+int main()
+{
+  std::vector<int> v= {1,2,3,4,5,6,7,8,9,10,11,12,13};
+
+  std::cout << " -> v                       = ";
+  doc_utils::print(v);
+
+  std::cout << " -> eve::views::reverse(v)  = ";
+  doc_utils::print(eve::views::reverse(v));
```

Comment:
There is an algo reduce that's faster than view reverse

Reply:
This file's purpose is to end up in eve::reverse documentation. So what do you want to do with eve::reduce?
If it's faster, why not just use it internally in eve::reverse ?

Reply:
algo::reverse I meant sorry

Reply:
Oh OK, makes sense now